code/_onclick/hud/robot.dm

```diff
@@ -2,7 +2,7 @@
 	..()
 	var/obj/screen/using
 
-//Radio
+	//Radio
```

Comment:
These sorts of formatting changes are also unnecessary, if someone wants to modify it when coding that's fine but we don't have style guidelines for where the comment needs to go.